Resources/Prototypes/Entities/Mobs/NPCs/silicon.yml

```diff
@@ -217,6 +217,7 @@
     flavorKind: station-event-random-sentience-flavor-mechanical
   - type: Absorbent
     pickupAmount: 10
+    useAbsorberSolution: true
```

Comment:
Isn't this pointless since its true by default?

Reply:
Just more verbal approach. Also having it as true by default is not quite usual stuff as bool is expected to be false by default. But yes, its not doing anything particular, just marking behaviour.